mars/deploy/oscar/local.py

```diff
@@ -268,11 +278,19 @@ def __init__(self: ClientType, cluster: ClusterType, session: AbstractSession):
 
     @classmethod
     async def create(
-        cls, cluster: LocalCluster, backend: str = None, timeout: float = None
+        cls,
+        cluster: LocalCluster,
+        backend: str = None,
+        execution_backend: str = None,
+        timeout: float = None,
     ) -> ClientType:
         backend = backend or "oscar"
         session = await _new_session(
-            cluster.external_address, backend=backend, default=True, timeout=timeout
+            cluster.external_address,
+            backend=backend,
```

Comment:
Now it's weird to see the `backend` here actually takes no effect, both Mars & ray pass the backend with value `oscar`, I wonder if we can unify the `backend` and `execution_backend`?

Reply:
So, We can resuse the backend for ray execution backend? e.g. 
- backend == 'oscar' for Mars & Mars on Ray
- backend == 'ray' for Mars on Ray DAG

Reply:
So, We can resuse the backend for ray execution backend? e.g. 
- backend == 'oscar' for Mars & Mars on Ray
- backend == 'ray' for Mars on Ray DAG

Reply:
Maybe just name mars for mars, ray for mars on ray?

Reply:
But, the `backend` is bound to the session cls. The execution backend only for the execution implementation. I am not sure if it is a good idea to mix them to one.

Currenty, 

- backend
    - oscar - _IsolatedSession
    - test - CheckedSession
- execution backend
    - mars - Mars, Mars on Ray
    - ray - Mars on Ray DAG

I can mix them to one,

- backend
    - mars - Mars, Mars on Ray
    - ray - Mars on Ray DAG

The backend will never bounds to the session cls.

Reply:
I think that's ok, backend for different backend not for session cls.

Reply:
I will push a commit to fix this. Thanks.